### PR TITLE
qa/tests: added new client-upgrade-*-nautilus suites for jewel, luminous, mimic

### PR DESCRIPTION
**Pull request by yuriw**
…

NOTE:
!!!! three suites below MUST use --suite-branch jewel, luminous, mimic (see https://tracker.ceph.com/issues/24021)
ref: https://github.com/ceph/ceph/pull/27983; https://github.com/ceph/ceph/pull/27934; https://github.com/ceph/ceph/pull/28027
--filter "ubuntu_16.04,centos_7.6,rhel_7.6" - test ONLY supported distro BEFORE mimic
--filter "ubuntu_16.04,ubuntu_18.04,centos_7.6,rhel_7.6" - test ONLY supported distro AFTER mimic

Merge after https://github.com/ceph/ceph/pull/28027

Signed-off-by: Yuri Weinstein <yweinste@redhat.com>

- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

### CODE REVIEW

File: qa/crontab/teuthology-cronjobs [hunk 1]
```diff
@@ -176,13 +176,14 @@ DISTRO_MIMIC="ubuntu_16.04,ubuntu_18.04,centos_7.4,rhel_7.5"
...
-## to run on ovh use ~/rhel_only_on_ovh.yaml, we will run on smithi nodes
-DISTRO_MIMIC="ubuntu_16.04,ubuntu_18.04,centos_7.4,rhel_7.5"
-47 01 * * 1,3  CEPH_BRANCH=nautilus; MACHINE_NAME=smithi; /home/teuthology/bin/cron_wrapper teuthology-suite -v -c $CEPH_BRANCH  -n 7 -m $MACHINE_NAME -s upgrade/client-upgrade-jewel -k distro -e $CEPH_QA_EMAIL --suite-branch jewel --filter $DISTRO_MIMIC
-50 01 * * 1,3  CEPH_BRANCH=nautilus; MACHINE_NAME=smithi; /home/teuthology/bin/cron_wrapper teuthology-suite -v -c $CEPH_BRANCH  -n 7 -m $MACHINE_NAME -s upgrade/client-upgrade-luminous -k distro -e $CEPH_QA_EMAIL --suite-branch luminous --filter $DISTRO_MIMIC
-
+## !!!! three suites below MUST use --suite-branch jewel, luminous, mimic (see https://tracker.ceph.com/issues/24021)
+## ref: https://github.com/ceph/ceph/pull/27983; https://github.com/ceph/ceph/pull/27934; https://github.com/ceph/ceph/pull/28027
+## --filter "ubuntu_16.04,ubuntu_18.04,centos_7.6,rhel_7.6" - test ONLY supported distro
+##
+## DISTRO_NAUTILUS="ubuntu_16.04,centos_7.6,rhel_7.6"
```
Comment (dillaman) [inline qa/crontab/teuthology-cronjobs]: Is this meant to be commented out? ```ubuntu_18.04``` can be added back for mimic->nautilus as well.

Comment (yuriw) [reply qa/crontab/teuthology-cronjobs]: @dillaman this is just for memory in crontab 
corrected

File: qa/crontab/teuthology-cronjobs [hunk 2]
```diff
@@ -176,13 +176,14 @@ DISTRO_MIMIC="ubuntu_16.04,ubuntu_18.04,centos_7.4,rhel_7.5"
...
+47 01 * * 1,3  CEPH_BRANCH=nautilus; MACHINE_NAME=smithi; DISTRO_NAUTILUS="ubuntu_16.04,centos_7.6,rhel_7.6"; /home/teuthology/bin/cron_wrapper teuthology-suite -v -c $CEPH_BRANCH  -n 7 -m $MACHINE_NAME -s upgrade/client-upgrade-jewel-nautilus -k distro -e $CEPH_QA_EMAIL --suite-branch jewel --filter $DISTRO_NAUTILUS
+50 01 * * 1,3  CEPH_BRANCH=nautilus; MACHINE_NAME=smithi; DISTRO_NAUTILUS="ubuntu_16.04,centos_7.6,rhel_7.6"; /home/teuthology/bin/cron_wrapper teuthology-suite -v -c $CEPH_BRANCH  -n 7 -m $MACHINE_NAME -s upgrade/client-upgrade-luminous-nautilus -k distro -e $CEPH_QA_EMAIL --suite-branch luminous --filter $DISTRO_NAUTILUS
+50 01 * * 1,3  CEPH_BRANCH=nautilus; MACHINE_NAME=smithi; DISTRO_NAUTILUS="ubuntu_16.04,centos_7.6,rhel_7.6"; /home/teuthology/bin/cron_wrapper teuthology-suite -v -c $CEPH_BRANCH  -n 7 -m $MACHINE_NAME -s upgrade/client-upgrade-mimic -k distro -e $CEPH_QA_EMAIL --suite-branch luminous --filter $DISTRO_NAUTILUS
```
Comment (dillaman) [inline qa/crontab/teuthology-cronjobs]: Nit: add ```ubuntu_18.04``` as well

Comment (yuriw) [reply qa/crontab/teuthology-cronjobs]: @dillaman bad commit, my bad
`mimic=>nautilus` runs with `DISTRO_NAUTILUS="ubuntu_16.04,centos_7.6,ubuntu_18.04,rhel_7.6"`

**Comment by yuriw** [comment]
testing after all supporting PRs merged :

http://pulpito.ceph.com/teuthology-2019-05-15_16:14:08-upgrade:client-upgrade-mimic-nautilus-distro-basic-smithi

http://pulpito.ceph.com/teuthology-2019-05-15_16:12:06-upgrade:client-upgrade-jewel-nautilus-nautilus-distro-basic-smithi

**Pull request merged**